.gitignore

```diff
@@ -45,4 +45,6 @@ yarn.lock
 public/robots.txt
 public/sitemap.xml
 public/sitemap-*.xml
-api.json
\ No newline at end of file
+api.json
+build/
+.next/
```

Comment:
I added these back in since infrequent users of this repo (like myself) probably have these artifacts lying around